codec/types/interface_registry.go

```diff
@@ -43,6 +46,16 @@ type InterfaceRegistry interface {
 	//  registry.RegisterImplementations((*sdk.Msg)(nil), &MsgSend{}, &MsgMultiSend{})
 	RegisterImplementations(iface interface{}, impls ...proto.Message)
 
+	// RegisterCustomTypeURL allows a protobuf service method request type to be
```

Comment:
This doc string should be more generic and reference service method requests as an example usage